src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Read [Java coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements), and see how we should organize the import statement :) Should you import the entire package, or only import classes that are exactly used?